### ISSUE TITLE
Selected state of the Nav bar is missing

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#911]
Original Auditor URL: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/ea12beac-7c82-11ec-92b3-770724cf4119?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=22&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

URL: https://cauldron.dequelabs.com/components/NavBar

## Description
The selected state of an element is not conveyed to screen reader users.

Element : Nav Item 3 

Note:
This issue is present in both mobile and non-mobile view
### RULE : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

### HOW TO FIX: 
Fix this issue by using the aria-current attribute to indicate which element is the current item within a container or set of related elements, such as the currently selected page, step, location, date, or time.

### REFERENCE: 
WAI-ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/#aria-current

### BACKGROUND: 
States and properties are attributes used to convey essential information about an element to screen readers and other assistive technologies. Some roles require certain state and property information - such as the checked/unchecked state of a checkbox. Native HTML elements provide those required states and properties, so nothing more needs to be done. If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add the relevant states and properties using ARIA.
